src/v2/Components/FlashBanner/__tests__/FlashBanner.jest.tsx

```diff
@@ -25,8 +65,82 @@ describe("FlashBanner", () => {
     const fakeLocation: any = { search: "?flash_message=expired_token" }
     window.location = fakeLocation
 
-    const wrapper = mount(<FlashBanner />)
+    const wrapper = mount(<BaseFlashBanner me={mockMe as any} />)
 
     expect(wrapper.text()).toContain("Link expired. Resend verification email.")
   })
+
+  describe("with relay", () => {
+    // // TODO: We dont need this test but for some reason it doesnt work. why???
+    // it("this test is broken???", async () => {
+    //   const wrapper = await getWrapper(
+    //     {
+    //       me: { id: "woot", canRequestEmailConfirmation: false },
+    //     },
+    //     "?flash_message=confirmed"
+    //   )
+    //   expect(wrapper.text()).toContain("Your email has been confirmed.")
+    // })
+    // describe("nested", () => {
+    //   describe("in another nest", () => {
+    //     // TODO: We dont need this test but for some reason it doesnt work. why???
+    //     it("but this identitcal test is not broken!!!", async () => {
```

Comment:
left this in for now because it's bothering me. output is
```
   fragment container
      ✕ this test is broken??? (123ms)
      nested
        in another nest
          ✓ but this identitcal test is not broken!!! (128ms)
```
with all other equally-nested tests passing fine.

Reply:
This is removed now but i'm still having occasional strange issues in similar cases.